promotion/suggestions.go

```diff
@@ -207,6 +210,16 @@ func (service *Service) Suggest(ctx context.Context, credentials []CredentialBin
 		return err
 	}
 
+	for _, funding := range fundingTypes {
+		value, _ := total.Float64()
```

Comment:
this seems inaccurate since we're counting the total contribution amount against funding source (rather than the amount that the funding source comprised of the total)